tests/benchmarks/current_consumption/system_off/Kconfig

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+mainmenu "Nordic SYSTEM_OFF demo"
+
+choice
+	prompt "Use retention"
+	optional
+
+config APP_USE_RETAINED_MEM
+	bool "Use state retention in system off using retained_mem driver"
+	depends on RETAINED_MEM
+
+endchoice
```

Comment:
why is this a choice? Just make it a normal Kconfig? Or get rid of it entirely and just check if RETAINED_MEM is enabled, don't see the point in another Kconfig...

Reply:
it is true that this definition is redundant, removed it